src/OrchardCore.Modules/OrchardCore.Widgets/Views/_ViewImports.cshtml

```diff
@@ -2,12 +2,13 @@
     For more information on enabling MVC for empty projects, visit http://go.microsoft.com/fwlink/?LinkID=397860
 
 *@
+@using OrchardCore.ContentManagement;
+@using OrchardCore.DisplayManagement.Views;
+@using Microsoft.AspNetCore.Authorization;
```

Comment:
why moving above taghelpers? most other places it is below?

Reply:
To be honest it's never the same everywhere but it feels more natural to add the using first to me.

Reply:
See https://github.com/OrchardCMS/OrchardCore/blob/dev/src/OrchardCore.Modules/OrchardCore.Deployment/Views/_ViewImports.cshtml
It doesn't really matter.
What matters is that I'm adding `@using Microsoft.AspNetCore.Authorization;`